.evergreen.yml

```diff
@@ -26,9 +29,15 @@ variables:
             -e EVERGREEN_EXPANSIONS_PATH=/tmp/build/tmp/expansions.yaml \
             -e NODE_JS_VERSION \
             -e BUILD_VARIANT \
+            -e AWS_ACCESS_KEY_ID -e AWS_SECRET_ACCESS_KEY -e SCCACHE_BUCKET \
```

Comment:
this is going to be done by `scripts/import-expansions.js`

Reply:
That wouldn’t load it under the right names, though (unless we also add these here to the project config)

Reply:
That's right, let's see how it goes then!